src/components/application_manager/test/commands/hmi/hmi_notifications/hmi_notifications_test.cc

```diff
@@ -896,7 +896,7 @@ TEST_F(HMICommandsNotificationsTest,
   utils::SharedPtr<Command> command =
       CreateCommand<OnDeviceStateChangedNotification>(message);
 
-  std::string device_id = "device_id";
+  std::string device_id = "AA15F2204D6B";
```

Comment:
@okozlovlux Could `"AA15F2204D6B"` be exposed as constant? Also could `device_id` be a `const std::string`?